client/js/components/cookie-notification.js

```diff
@@ -0,0 +1,32 @@
+import cookie from 'cookie-cutter';
+const COOKIE_NAME = 'cookies_accepted';
+
+const cookieNotification = (el) => {
+  const isAccepted = cookie.get(COOKIE_NAME);
+  const closeButton = document.getElementById('cookie-notification-close');
+
+  if (isAccepted) {
+    el.parentNode.removeChild(el);
+
+    return;
+  }
+
+  el.classList.add('cookie-notification--is-visible');
+
+  const setCookie = () => {
```

Comment:
Why not have this as a function outside of the init, or just attach it anonymously to the event handler?